src/util/Helpers.cpp

```diff
@@ -316,6 +324,50 @@ QLocale getSystemLocale()
     return QLocale::system();
 }
 
+void writeProviderEmotesCache(const QString &id, const QString &provider,
+                              const QByteArray &bytes)
+{
+    std::ignore = QtConcurrent::run([bytes, id, provider]() {
```

Comment:
Please use [`QThreadPool::globalInstance()->start(callable)`](https://doc.qt.io/qt-6/qthreadpool.html#start-1), because we don't need the result of this.